libraries/libdxfrw/src/drw_objects.h

```diff
@@ -375,6 +375,31 @@ class DRW_Textstyle : public DRW_TableEntry {
     int fontFamily;         /*!< ttf font family, italic and bold flags, code 1071 */
 };
 
+class DRW_UCS:public DRW_TableEntry{
+    SETOBJFRIENDS
+public:
+    DRW_Coord origin;
+    DRW_Coord xAxisDirection;
+    DRW_Coord yAxisDirection;
+    DRW_Coord orthoOrigin;
+    double elevation;
+    int orthoType;
+
+    DRW_UCS() { reset();}
```

Comment:
using default member values for trivial values

a special init() method to indicate non-trivial initialization.

reset() member may not necessary, since you can always assign it to a new variable:
```C++
DRW_UCS ucs{};
// after some usage
ucs = {}; // instead of ucs.reset();
```
I found less logic makes the code easier to read. For example, a novice developer may not need to find what reset() actually does.